linux/devdoc/threadpool_linux_requirements.md

```diff
@@ -0,0 +1,258 @@
+# threadpool_linux
+
+## Overview
+
+`threadpool_linux` is the Linux-specific implementation of the `threadpool` module.
```

Comment:
It would be nice to explain a bit in the overview WHAT we want this unit to do. For example: The main purpose of the unit is to schedule work items that are executed on a number of predefined threads.

Reply:
added